src/main/remote-sync/helpers.ts

```diff
@@ -0,0 +1,86 @@
+import Store from 'electron-store';
+
+export const remoteSyncStore = new Store<{
+  lastFilesSyncAt?: string;
+  lastFoldersSyncAt?: string;
+}>();
+
+export const clearRemoteSyncStore = () => remoteSyncStore.clear();
+export function getLastFilesSyncAt(): Date | undefined {
+  const value = remoteSyncStore.get('lastFilesSyncAt');
+
+  if (!value) return undefined;
+
+  return new Date(value);
+}
+
+export function saveLastFilesSyncAt(date: Date, offsetMs: number): Date {
+  remoteSyncStore.set(
+    'lastFilesSyncAt',
+    new Date(date.getTime() - offsetMs).toISOString()
+  );
+  return date;
+}
+
+export function getLastFoldersSyncAt(): Date | undefined {
+  const value = remoteSyncStore.get('lastFoldersSyncAt');
+
+  if (!value) return undefined;
+
+  return new Date(value);
+}
+
+export function saveLastFoldersSyncAt(date: Date, offsetMs: number): Date {
+  remoteSyncStore.set(
+    'lastFoldersSyncAt',
+    new Date(date.getTime() - offsetMs).toISOString()
+  );
+  return date;
+}
+
+export type RemoteSyncedFile = {
+  id: number;
+  uuid?: string;
+  fileId: string;
+  type: string;
+  size?: string;
+  bucket?: string;
```

Comment:
This always exists, could be null, but the field exists always, is not undefined